packages/dev-tools-pages/ts/components/Animations/Compiler/index.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
+
+import Animation from '../index';
+import * as animationData from './data.json';
+
+function AnimationCompiler() {
```

Comment:
convention: for stateless components can we use `const AnimationCompiler: React.StatelessComponent<AnimationCompilerProps> = () => ()` syntax